### PR TITLE
Use tslint --test CLI command for simpler testing

### PR DESCRIPTION
**Pull request by JKillian**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,11 +11,11 @@
     "build": "npm-run-all -p compile lint",
     "clean": "rm -rf build/",
     "compile": "tsc -p ./",
-    "copy": "cp -f package.json build/src/rules/",
+    "copy": "cp -f package.json build/",
     "lint": "tslint 'src/**/*.ts'",
     "prepublish": "npm-run-all clean test copy",
     "test": "npm-run-all build test-rules",
-    "test-rules": "node ./build/test/ruleTestRunner.js"
+    "test-rules": "for path in ./test/rules/*; do tslint -r ./build/rules --test $path; done"
```
Comment (adidahiya) [inline package.json]: this is neat but also kind of scares me that you can _write bash in package.json_. :shrug:

File: package.json [hunk 2]
```diff
@@ -11,11 +11,11 @@
     "build": "npm-run-all -p compile lint",
     "clean": "rm -rf build/",
     "compile": "tsc -p ./",
-    "copy": "cp -f package.json build/src/rules/",
+    "copy": "cp -f package.json build/",
```
Comment (adidahiya) [inline package.json]: I think you need to adjust the publish script to account for this change as well.

**Comment by adidahiya** [comment]
👎

**Comment by JKillian** [comment]
Modified everything so that the built version of the rules get placed directly in `build/` which is also the directory that is published. The `package.json`, `tsconfig.json`, and publish script should all by in sync now

**Pull request merged**